BaikalStandalone/Application/app_utils.cpp

```diff
@@ -205,6 +205,10 @@ namespace Baikal
             {
                 s.denoiser_type = DenoiserType::kML;
             }
+            else if (denoiser_type == "rif")
```

Comment:
switch ... case ?

Reply:
It doesn't work with anything but integers in C++ :)